tests/toranj/openthread-core-toranj-config-simulation.h

```diff
@@ -83,4 +83,12 @@
  */
 #define OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE 1
 
+/**
+ * @def OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE
+ *
+ * Define to 1 to enable power calibration support.
+ *
+ */
+#define OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE 1
```

Comment:
Just wondering why enabling it in the `toranj` test config? Is it to validate that it builds ok?

Reply:
If we do not enable it here, building the unit test of the power calibration will fail.

Reply:
I have added the `#if OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE ` to the unit test of the power calibration. It doesn't need to add the config here.